packages/fxa-content-server/app/scripts/templates/partial/brand-messaging.mustache

```diff
@@ -9,28 +9,28 @@
                 </div>
             </div>
             <div class="flex-initial max-w-md pr-2">
-                <p class="text-left text-sm font-bold">
+                <p class="text-left text-sm font-bold w-5/6 mobileLandscape:w-full">
                     {{#t}}Firefox accounts will be renamed Mozilla accounts on Nov 1{{/t}}
                 </p>
-                <p class="text-left text-xs">
+                <p class="text-left text-xs w-5/6 mobileLandscape:w-full">
                     {{#t}}You’ll still sign in with the same username and password, and there are no other changes to the products that you use.{{/t}}
-                    <span class="brand-learn-more cursor-pointer underline">{{#t}}Learn more{{/t}}</a>
+                    <span role="link" class="brand-learn-more cursor-pointer underline">{{#t}}Learn more{{/t}}</a>
                 </p>
             </div>
         </div>
         {{/showPrelaunch}}
         {{#showPostlaunch}}
         <div className="flex"/>
             <div>
-                <p class="text-sm font-bold">
+                <p class="text-sm font-bold w-5/6 mobileLandscape:w-full">
                     {{#t}}We’ve renamed Firefox accounts to Mozilla accounts. You’ll still sign in with the same username and password, and there are no other changes to the products that you use.{{/t}}
                     <span role="link" class="brand-learn-more cursor-pointer underline">{{#t}}Learn more{{/t}}</a>
                 </p>
             </div>
         </div>
         {{/showPostlaunch}}
     </div>
-    <div class="absolute w-10 right-0 top-0 pt-3 start-0">
+    <div class="absolute w-10 top-0 rtl:left-0 ltr:right-0 pt-3">
```

Comment:
Missed this in first review, but this should be a `button` instead of a `div` since it's interactive and should be tabbable.

Also, I'm not sure what's causing this, but in the Backbone version, I can't seem to close this now? I restarted content after checking into your branch, hit `/clear`, and hitting `/` and trying to click the X it does not dismiss. 🤔

Reply:
I've changed the CSS a bit, so this is shouldn't be a problem anymore. I did see this occasionally due to  something overlapping this area. I think now that it's using flexbox it won't happen, but would be curious if you can reproduce.